### ISSUE TITLE
Registration successful box is not WCAG-compliant

### ISSUE THREAD

**Issue by bettytro** [NoroffFEU/agency.noroff.dev#648]
The green registration successful box is not WCAG-compliant because it shows green text on a green box. Especially the link to login is almost impossible to read. 

Text color should be changed to a more readable color.

**Comment by AnaCascante** [comment]
@Sugal99 is this issue solved? if solved, the issue can be closed
